schema/bom-1.6.proto

```diff
@@ -171,16 +171,12 @@ enum DataFlowDirection {
 }
 
 message Dependency {
-  enum DependencyType {
-    DEPENDENCY_TYPE_IMPLEMENTS = 0;
-    DEPENDENCY_TYPE_USES = 1;
-  }
   // References a component or service by the its bom-ref attribute
   string ref = 1;
   // The bom-ref identifiers of the components or services that are dependencies of this dependency object.
   repeated Dependency dependencies = 2;
-  // Defines and characterizes the type of dependency
-  optional DependencyType type = 3;
+  // The bom-ref identifiers of the components or services that define a given specification or standard, which are provided or implemented by this dependency object.
+  repeated string provides = 3;
```

Comment:
`provides` is a verb. Can we use a bit generic like `capabilities` to indicate a state and not an action?

Reply:
we used "depends on" on the sibling ...

Reply:
> `provides` is a verb. Can we use a bit generic like `capabilities` to indicate a state and not an action?

for protobuff, I'd agree. a noun would go with the current naming conventions.

for JSON/XML, the current naming conventions are verbs.

Reply:
Sounds like your suggestion for `capabilities` @prabhu would be a good fit for what we're doing with CDX 1.7 - specifically architecture and blueprints.